### DISCUSSION TITLE
How to track a sat with inscription?

### DISCUSSION THREAD

**Discussion by pengux** [ordinals/ord#2312] [Q&A]
How do I know when a sat has been traded and which address owns it currently? Is it possible to find out through the block explorer?

**Comment by gkmz** [comment]
This question bothers me just as much. 

I know that inscriptions are inscribed via Taproot's verification script, which means that the inscription data encoded and saved in the witness of input of UTXO, but if the UTXO is spent, transferring the inscription and the sats that correspond to it, how can the inscription be tracked through those sats? I know it can be tracked using ord software, but my question is, is it possible to track the inscription of sats after transfering multi times on the bitcoin chain without ord? If not, does that mean that ord is recording the relationship between the sats and the inscriptions through a local database? Thank you for your answers.

**Comment by bugodx** [comment]
I'm not sure, but if track inscription data in sats through the bitcoin chain, it should require a lot of queries and need to go through the transaction data to track down the original inscription data on chain, right? How does ord do this?
